platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -693,6 +698,21 @@
 		DA8848331CBAFB2A00AB86E3 /* Kit */ = {
 			isa = PBXGroup;
 			children = (
+				4018B1C31CDC277F00F666AF /* MGLAnnotationView_Private.h */,
+				4018B1C41CDC277F00F666AF /* MGLAnnotationView.mm */,
+				4018B1C51CDC277F00F666AF /* MGLAnnotationView.h */,
+				404349CA1CF66B3E00FAA2CA /* MGLAnnotationContainerView.h */,
+				404349CB1CF66B3E00FAA2CA /* MGLAnnotationContainerView.m */,
+				DA8848341CBAFB8500AB86E3 /* MGLAnnotationImage.h */,
+				DA8848401CBAFB9800AB86E3 /* MGLAnnotationImage_Private.h */,
+				DA8848411CBAFB9800AB86E3 /* MGLAnnotationImage.m */,
+				DA8848421CBAFB9800AB86E3 /* MGLAPIClient.h */,
+				DA8848431CBAFB9800AB86E3 /* MGLAPIClient.m */,
+				DA8848351CBAFB8500AB86E3 /* MGLCalloutView.h */,
+				DA8848441CBAFB9800AB86E3 /* MGLCompactCalloutView.h */,
+				DA8848451CBAFB9800AB86E3 /* MGLCompactCalloutView.m */,
+				DA8848461CBAFB9800AB86E3 /* MGLLocationManager.h */,
+				DA8848471CBAFB9800AB86E3 /* MGLLocationManager.m */,
```

Comment:
This is a bad merge. I moved most of these files to the Annotations group. You should move MGLAnnotationContainerView.h/.m to that group but otherwise omit this diff.